apps/desktop/src/lib/gitHost/gitlab/gitlab.ts

```diff
@@ -1,13 +1,14 @@
-import { GitLabBranch } from './gitlabBranch';
 import type { RepoInfo } from '$lib/url/gitUrl';
 import type { GitHost } from '../interface/gitHost';
 import type { DetailedPullRequest, GitHostArguments } from '../interface/types';
+import { GitLabBranch } from './gitlabBranch';
 
 export type PrAction = 'creating_pr';
 export type PrState = { busy: boolean; branchId: string; action?: PrAction };
 export type PrCacheKey = { value: DetailedPullRequest | undefined; fetchedAt: Date };
 
 export const GITLAB_DOMAIN = 'gitlab.com';
+export const GITLAB_SUB_DOMAIN = 'gitlab'; // For self hosted instance of Gitlab
```

Comment:
Well this now assumes that folks are putting their Gitlab instance at `gitlab.company.com` always, right?

What do you think about makign this a user setting? @mtsgrd

Reply:
I was thinking this is a change in the right direction, even if we want to make it explicitly configurable? Aside from that, I do think we should start prioritising integration work, both building out our GH support, as well as building real integrations for the rest.

Reply:
> Well this now assumes that folks are putting their Gitlab instance at gitlab.company.com always, right?

Yes, this is one issue. The better solution would be to have some setting where the user could define this in case the git url is different.